components/User/TopUserList.tsx

```diff
@@ -0,0 +1,157 @@
+import { Row, Col, Table, Form, Button, Image, Badge } from 'react-bootstrap';
+import { TopUser } from '../../models/User';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEnvelope } from '@fortawesome/free-solid-svg-icons';
+
+import { faGithub } from '@fortawesome/free-brands-svg-icons';
+
+import styles from '../../styles/TopUserList.module.less';
+export interface TopUserListProps {
+  value: TopUser[];
+}
+
+const parseFn = (str: string) => (str ? JSON.parse(str) : {});
+export class TopUserList {
+  static Layout = ({ value = [] }: TopUserListProps) => (
```

Comment:
```suggestion
export const TopUserList = ({ value = [] }: TopUserListProps) => (
```
没继承 `ScrollList` 抽象组件（无上拉加载更多）就没必要这么写，完全依赖服务端渲染的直接写成函数组件即可。